python/lsst/sconsUtils/utils.py

```diff
@@ -136,6 +136,7 @@ def libraryLoaderEnvironment():
 #  @brief Safe wrapper for running external programs, reading stdout, and sanitizing error messages.
 #
 #  Note that the entire program output is returned, not just a single line.
+# Returns strings not bytes
```

Comment:
Ah, I see what you mean. Doxygen @return. Ok, that is used elsewhere in sconsUtils.